site.cfg.example

```diff
@@ -130,6 +130,12 @@
 # include_dirs = /opt/OpenBLAS/include
 # runtime_library_dirs = /opt/OpenBLAS/lib
 
+# [blis]
+# libraries = blis
+# library_dirs = /home/username/blis/lib
+# include_dirs = /home/username/blis/include/blis
```

Comment:
hm but it seems where blis puts its stuff, should probably have a comment that this needs to be the folder containing cblas.h which is `prefix/blis` by default